### ISSUE TITLE
Failed GET request due to Minecraft's packaged runtime version

### ISSUE THREAD

**Issue by SnowShock35** [GenDeathrow/MPUtils#9]
Hi,

I believe to have found a bug with MPUtils / MPUtils Basic Info. I've stated all the information I have on the information bellow and would really appreciate it if you had the time to take a look at the issue I'm having. It's a rather unusual one so I don't see it being the easiest to debug or solve, but for references sake, I'd thought it best to report it anyway.

### Key Information 

- Minecraft 1.10.2
- Minecraft Forge 12.18.3.2254
- ModPack Utilities 1.2.5
- ModPack Basic Tools 1.1.5

### Description of Issue

After adjusting the configuration file to request my change log from the following URL https://content.blamesnow.co.uk/changelogs/sgp-foundation.txt, enabling HTTP and then opening up the game to view it from the games' menu only to find that the change log doesn't show and that I'm only greeted with the message `File Not Found`

### What Should Happen

I should load up the game after adjusting the configuration file and go to the change log GUI and see my change log appear

### What Actually Happens

The change log fails to show up and instead I just see `File Not Found`

### The Cause

The cause seems to be with run time that Minecraft is packaged with, or how you're handling the GET request of the file and it not being compatible with that specific release of Java. Regardless, the issue is with handling GET requests using the HTTPS protocol and failing to GET the file when that specific protocol is present in the specified path URL of the change log. HTTP by itself is fine however, secure HTTP is not. 

From the testing I've tone, it seems that this is not the case with the newest available release of java 1.8.0_121 and since I've been using that version for so long I've yet to see this issue. However, now that support for the jar launcher is being dropped (For Twitch App at least) this will become a more prominent issue in the coming months.

### Issue Media

Here is the error that was present in `latest.log` when the issue occurred.

http://pastebin.com/Yi7sMaSg

**Comment by GenDeathrow** [comment]
I looked into the twitch app dropping the jar launcher. Its true, but you still have access to changing the version of java minecraft runs on just like the current jar launcher is. TBH ppl need to be on the latest version of java anyways, as a lot of mods are using java 8 and no longer support previous versions. I have normally tried to support older versions, but new java 8 methods have gotten to be a lot better. If you have a link about the twitch app doing something, than please link it. But from what I found mojang is just changing their launcher.  

Now I'm technically using proper methods for calling URL, but not sure why older versions don't work with secure. I couldn't find anything that would prevent secure not working at least. Most I can do is maybe leave a note in config ,or if a no file is found have a pop up to make sure players on latest version of java if they use the url. I appreciate ya finding this, I dont think I would have found this issue only because Im on the latest version of java. It wouldn't have been something I tested. So I grateful you figured it out. 

Edit: actually mputils is prolly going to get a java checker added to it... basically my old java checker mod is getting added into mputils.

**Comment by GenDeathrow** [comment]
By the way, one of the best bug reports.

**Comment by SnowShock35** [comment]
I had a feeling this would be the case. I'll most likely stick to jar for now while I can then when it's removed try and force latest java. Thanks for the feedback though. 🙂

**Comment by GenDeathrow** [comment]
Yeah from what ive been told by Modpack developers. That the biggest issue, they get a lot of reports of ppl on old java versions and it crash out.

**Comment by SnowShock35** [comment]
Yeah it's pretty common. :/
